Project.toml

```diff
@@ -67,6 +68,7 @@ Measures = "0.3"
 Optimization = "3, 4"
 OptimizationBBO = "0.3"
 OptimizationNLopt = "0.2"
+Plots = "<0.0.1, 1"
```

Comment:
Why not just Plots `1`? The `<0.0.1` syntax should be used only for standard libraries.